.github/workflows/pre-release.yml

```diff
@@ -0,0 +1,146 @@
+name: Publish Pre Release
+
+on:
+  push:
+    branches:
+#      - alpha
+      - feat/publishing # for testing
+
+jobs:
+  changelog:
+    name: Generate Changelog
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.changelog.outputs.tag }}
+      skipped: ${{ steps.changelog.outputs.skipped }}
+      clean_changelog: ${{ steps.changelog.outputs.clean_changelog }}
+      changelog: ${{ steps.changelog.outputs.changelog }}
+      build_number: ${{ steps.extract.outputs.build_number }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.PRIVATE_TOKEN }}
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - run: npm install conventional-changelog-conventionalcommits
+
+      - name: Create changelog
+        id: changelog
+        uses: DevScyu/conventional-changelog-action@v4
```

Comment:
Is this an action you've written yourself, or a modified fork from some upstream project? Why not use upstream if that is the case? Otherwise, if we really need a special tool or a modified version (I hoghly recommend against it -- even more maintenance burden!), it should move into the Wynntils area on github.

Reply:
This is a fork of https://github.com/TriPSs/conventional-changelog-action with changes to support pre-releases.

I have submitted a PR upstream here: https://github.com/TriPSs/conventional-changelog-action/pull/186